dotnet/src/webdriver/Interactions/InputDevice.cs

```diff
@@ -67,10 +65,7 @@ public string DeviceName
         /// Creates a pause action for synchronization with other action sequences.
         /// </summary>
         /// <returns>The <see cref="Interaction"/> representing the action.</returns>
-        public Interaction CreatePause()
-        {
-            return this.CreatePause(TimeSpan.Zero);
-        }
+        public Interaction CreatePause() => this.CreatePause(TimeSpan.Zero);
```

Comment:
Not for method, topic for editorconfig.

Reply:
Do you think we need a consistent rule here? I like to use method expressions if the method contains no real logic, just alternate parameters or for a wrapper.

If we want to always use method bodies, I can revert this change.

Reply:
Yes please revert, for methods. This question is out of scope of this PR.

Reply:
>  I like to use method expressions if the method contains no real logic, just alternate parameters or for a wrapper.

I like to preserve a placeholder via `{ }` to add more code inside when a method becomes to not `if the method contains no real logic`.

It is more for internal conventions (editorconfig topic), at this time let us be less destructive.

Reply:
I completely get that, it allows for much smaller diffs. I'll try to stick to that convention. The editorconfig PR is updated with the latest thinking btw :)